templates/project/template.nix

```diff
@@ -0,0 +1 @@
+{description = "Project environment template";}
```

Comment:
Are we going to teach users that there are two types of project environments, both in pkgs and rootlevel? If so what language are we going to use to differentiate?

Reply:
First we are going to teach about simple use case (one repository, one environment, rootlevel) and than we will provide a tutorial for growing this to a more advance use case (eg. monorepo setup, multiple packages/env in one repo... ).

The current documentation doesn't reflect that, but hopefully soon we will be able to show even nicer onboarding story.